src/patternfly/components/DataList/docs/data-list-simple-h2.md

```diff
@@ -0,0 +1,2 @@
+### Usage
+When a list item includes more than one block of content, it can be difficult for some screen reader users to discern where one list item ends and the next one begins. A simple way to provide better separation of list items is to define the primary content section as a heading. Headings are useful for indicating a new section of contents, but also provide an easy way for screen reader users to navigate to specific sections on the page. The heading level should be based on the context in which the DataList component is being used. For example, if the heading for the section that contains the DataList is a level 3, then `h4` elements should be used in the DataList list items.
```

Comment:
@jgiardino in `When a list item includes more than one block of content` do you mean when a list includes more than one list item?

Reply:
@jgiardino ^

Reply:
For example, this `<li>` includes two separate `<div>`s. When a `<li>` includes more than one block of content, it might not be obvious which blocks are grouped together inside a list item. Some screen readers like VO announce the beginning/end of a list item, but for some screen readers, this is not announced. Using headings is an easy way to organize the information in a way that's more obvious in some screen readers.

```
<li class="pf-c-data-list__item" aria-labelledby="simple-item1">
    <div class="pf-c-data-list__item-row">
      <div class="pf-c-data-list__item-content">
        <div class="pf-c-data-list__cell">
          <span id="simple-item1">Primary content</span>
        </div>
        <div class="pf-c-data-list__cell">
          Secondary content
        </div>
      </div>
    </div>
  </li>
```